front/src/App.tsx

```diff
@@ -147,7 +147,7 @@ function App() {
                   component={<Dashboard />}
                   redirectPath={prevLocation}
                   minBeta={3}
-                  requiredAbps={[["view_inventory", "view_shipments", "view_beneficiary_graph"]]}
+                  requiredAbps={["view_inventory", "view_shipments", "view_beneficiary_graph"]}
```

Comment:
the `[[ ... ]]` is indeed required. we want to enable the route if only one of the ABPs is present in the user's auth data